CMakeModules/CPackConfig.cmake

```diff
@@ -101,5 +101,9 @@ SET(CPACK_SOURCE_IGNORE_FILES
 # Ignore build directories that may be in the source tree
 FILE(GLOB_RECURSE CACHES "${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 
+SET(CPACK_WIX_LICENSE_RTF "${PROJECT_SOURCE_DIR}/LICENSE.rtf")
+SET(CPACK_WIX_PATCH_FILE "${PROJECT_SOURCE_DIR}/wix/WIXPatch.wxs")
+SET(CPACK_WIX_UPGRADE_GUID "FF9E2D77-CDC7-4D24-8B7B-99D66EDEE862")
```

Comment:
It's a unique identifier for the installer, used for when users upgrade to a newer version.  The existing installation can be detected by this.  This way multiple entries in Programs and Features aren't created, and the existing installation can be upgraded in place.

For more information, see: https://cmake.org/cmake/help/v3.6/module/CPackWIX.html